### DISCUSSION TITLE
Does open-webui's Image Generation feature support Azure DALL-E?

### DISCUSSION THREAD

**Discussion by changchiyou** [open-webui/open-webui#1244] [Q&A] [answered]
I can't find the right setting for what I need:

<img width="724" alt="image" src="https://github.com/open-webui/open-webui/assets/46549482/35da8531-771b-4c0d-be3c-66c026fb4494">

**Comment by justinh-rahb** [comment, marked as answer]
Not at present.

**Reply by changchiyou** [reply]
sad 😢

**Reply by changchiyou** [reply]
@justinh-rahb Is there any workaround for using litellm?
